package.json

```diff
@@ -1,30 +1,24 @@
 {
   "name": "pioneer-studies-addon-utils",
-  "description": "A utility library for Pioneer studies",
   "version": "1.0.0",
-  "maintainers": [
-    {
-      "name": "Rehan Dalal",
-      "email": "rdalal@mozilla.com"
-    }
-  ],
-  "repositories": [
-    {
-      "type": "git",
-      "url": "https://github.com/mozilla/pioneer-studies-addon-utils.git"
-    }
-  ],
-  "licenses": [
-    {
-      "name": "MPL 2.0",
-      "url": "https://github.com/mozilla/pioneer-studies-addon-utils/blob/master/LICENSE"
-    }
-  ],
-  "scripts": {
-    "build": "webpack"
-  },
+  "description": "A utility library for Pioneer studies",
+  "dependencies": {},
   "devDependencies": {
-    "jose-jwe-jws": "0.1.5",
+    "jose-jwe-jws": "git+https://github.com/square/js-jose.git",
```

Comment:
Why is this using a git url instead of pulling it from npm?

Reply:
so the latest release does not have some changes needed to allow us to set crypto without it being a global variable. this was fixed 24 days ago in master and I've requested that they create a new release but until then I have this using Jose right from git.